server/graphql/types/src/types/user.rs

```diff
@@ -84,6 +88,24 @@ impl UserNode {
             nodes,
         }
     }
+
+    pub async fn permissions(
+        &self,
+        ctx: &Context<'_>,
+        store_id: String,
```

Comment:
store_id could be optional can't it? if not specified you could query permissions for all stores... However, the user would have to remember to query for the store_id in the UserPermissionNode otherwise you could get duplicates in the result...

Which lets me think about a UserStorePermissionNode instead of UserPermissionNode:

```rust
#[Object]
impl UserStorePermissionNode {
    /// Returns all permissions for this store
    pub async fn permissions(&self) -> Vec<UserPermissionNodePermission> {
        
    }

    pub async fn store_id(&self) -> &Option<String> {
        &self.row().store_id
    }
}
```
This would also avoid redundantly returning the store id:
![image](https://user-images.githubusercontent.com/88299195/184025773-5830ca02-9818-4c53-98e5-09752b4971ef.png)